packages/select/src/FilterMultiSelect/components/MultiSelectOption/MultiSelectOption.tsx

```diff
@@ -45,8 +45,8 @@ export const MultiSelectOption = ({
         isFocusVisible ? styles.isFocusVisible : "",
         isDisabled ? styles.isDisabled : ""
       )}
-      aria-label={item.value.label}
-      aria-describedby={item.value.count ? countElementId : undefined}
+      aria-label={item.value?.label || ""}
```

Comment:
the generic Node<ItemType> can be null so this is required as of the updated types